mlir/include/mlir/Dialect/LLVMIR/GENXOps.td

```diff
@@ -67,10 +68,10 @@ def GENX_Dialect : Dialect {
     /// The address space value that represents private memory.
     static constexpr unsigned kPrivateMemoryAddressSpace = 0;
   }];
-
-  let useDefaultAttributePrinterParser = 1;
```

Comment:
We need a custom parser to handle the attributes in the new MatrixLoad operator

Reply:
Probably can add this back and remove `parseAttribute` and `printAttribute`. It works for me, give it a try.

Reply:
done.